horace_core/symop/@SymopReflection/SymopReflection.m

```diff
@@ -0,0 +1,127 @@
+classdef SymopReflection < Symop
+
+    properties(Dependent)
+        u;
+        v;
+        normvec;
+    end
+
+    properties(Access=private)
+        u_;
+        v_;
+    end
+
+    methods
+        function obj = SymopReflection(u, v, offset)
+            if nargin == 0
+                return
+            end
+
+            if ~exist('offset', 'var')
+                offset = obj.offset;
+            end
+
+            if ~SymopReflection.check_args({u, v, offset})
+                error('HORACE:symop:invalid_argument', ...
+                      ['Constructor arguments should be:\n', ...
+                       '- Reflection: symop(3vector, 3vector, [3vector])\n', ...
+                       'Received: %s'], disp2str({u, v, offset}));
+            end
+
+            if norm(cross(u, v)) < 1e-2
```

Comment:
could debate about size for this tolerance in absolute terms. But would 
```
un = u/norm(u)
vn = v/norm(v)
if norm(cross(un,vn))< whatever
```
be a safer option

Reply:
Done